Pricing/ItemPriceCalculator.php

```diff
@@ -50,23 +50,28 @@ public function calculate($item, $currency = null, $useProductsPrice = true)
         // get bulk price
         if ($useProductsPrice) {
             $product = $item->getCalcProduct();
-            $price = $this->priceManager->getBulkPriceForCurrency($product, $item->getCalcQuantity(), $currency);
-
-            // no price set - return 0
-            if ($price === null) {
-                return 0;
+            $specialPrice = $this->priceManager->getSpecialPriceForCurrency($product, $currency);
+            if (!empty($specialPrice)) {
+                $priceValue = $specialPrice->getPrice();
+            } else {
```

Comment:
Please also consider that a bulk price can be smaller than the special price. So you need to check the special price and the bulk price and take the smaller of both.